docs/usage.md

```diff
@@ -6,51 +6,28 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+The input data can be passed to nf-core/proteinfamilies using the `--input` parameter of amino acid fasta files.
```

Comment:
```suggestion
**nf-core/proteinfamilies** is a bioinformatics pipeline that generates protein families from amino acid sequences and/or updates existing families with new sequences.
It takes a protein fasta file as input, clusters the sequences and then generates protein family Hiden Markov Models (HMMs) along with their multiple sequence alignments (MSAs).
Optionally, paths to existing family HMMs and MSAs can be given (must have matching base filenames one-to-one) in order to update with new sequences in case of matching hits.
```

I don't really get this sentence. I recommend just relplacing as what teh pipeline does.